AUTHORS

```diff
@@ -1050,3 +1050,4 @@ Rodrigo Luger <rodluger@gmail.com>
 Marco Antônio Habitzreuter <mahabitzreuter@gmail.com>
 Ayush Bisht <bisht.ayush2001@gmail.com>
 Akshansh Bhatt <akshansh@tuta.io>
+migueltorrescosta <miguelptcosta1995@gmail.com>
```

Comment:
This file is automatically generated from `./bin/authors_update.py`. If you want to add yourself, the best thing you can do would be to add an entry in .mailmap mapping your full name, and your name will be added here the next time we run this script (you can also run the script yourself but it hasn't been run in a while so a bunch of other people's entries will need to be adjusted if you do).

Reply:
On it. I ran `./bin/authors_update.py` first, but as it added a bunch of people it felt like it wasn't doing what was expected